src/createXcmTypes/RelayToSystem.ts

```diff
@@ -148,14 +148,24 @@ export const RelayToSystem: ICreateXcmType = {
 	 * Create a WeightLimitV2 type.
```

Comment:
If we use `XcmV3WeightLimit`, lets make sure to change the header comment to reflect that.

Reply:
Same with all the other `createWeightLimit` changes.